dev-tools/packaging/packages.yml

```diff
@@ -411,3 +411,11 @@ specs:
         files:
           '{{.BeatName}}{{.BinaryExt}}':
             source: ./{{.XPackDir}}/{{.BeatName}}/build/golang-crossbuild/{{.BeatName}}-{{.GOOS}}-{{.Platform.Arch}}{{.BinaryExt}}
```

Comment:
Is this `files` override also needed in the new spec?

Reply:
@mikemadden42 Is there a reason you didn't address this issue? Can you please verify that these three new images have the appropriate x-pack binary included in them (and not the OSS one).